com.endlessm.HackSoundServer.json.in

```diff
@@ -0,0 +1,27 @@
+{
+    "app-id": "com.endlessm.HackSoundServer",
```

Comment:
Should have a `"branch": "@BRANCH@"` in here (I discovered this on Friday while integrating the hackdex with Jenkins.)